website/source/docs/platform/k8s/helm.html.md

```diff
@@ -0,0 +1,259 @@
+properly installed and configured with your Kubernetes cluster.
+
+-> **Important:** The Helm chart is new and
+may still change significantly over time. Please always run Helm with
+`--dry-run` before any install or upgrade to verify changes.
+
+## Using the Helm Chart
+
+To use the Helm chart, you must download or clone the
+[consul-helm GitHub repository](https://github.com/hashicorp/consul-helm)
+and run Helm against the directory. We plan to transition to using a real
+Helm repository soon. When running Helm, we highly recommend you always
+checkout a specific tagged release of the chart to avoid any
+instabilities from master.
+
+Prior to this, you must have Helm installed and configured both in your
+Kubernetes cluster and locally on your machine. The steps to do this are
+out of the scope of this document, please read the
+[Helm documentation](https://helm.sh/) for more information.
+
+Example chart usage:
+
+```sh
+# Clone the chart repo
+$ git clone https://github.com/hashicorp/consul-helm.git
+$ cd consul-helm
+
+# Checkout a tagged version
+$ git checkout v0.1.0
+
+# Run Helm
+$ helm install --dry-run ./
+```
+
+~> **Warning:** By default, the chart will install _everything_: a
+Consul server cluster, client agents on all nodes, feature components, etc.
+This provides a nice out-of-box experience for new users, but may not be
+appropriate for a production setup. Considering setting the `global.enabled`
+value to `false` and opt-in to the various components.
+
+## Configuration (Values)
+
+The chart is highly customizable using
+[Helm configuration values](https://docs.helm.sh/using_helm/#customizing-the-chart-before-installing).
+Each value has a sane default tuned for an optimal getting started experience
+with Consul. Before going into production, please review the parameters below
+and consider if they're appropriate for your deployment.
+
+* <a name="v-global" href="#v-global">`server`</a> - These values are global values that affect multiple components of the chart.
+
+  - <a name="v-global-enabled" href="#v-global-enabled">`enabled`</a> (`boolean: true`) -
+  The master enabled/disabled configuration. If this is true, most components
+  will be installed by default. If this is false, no components will be installed
+  by default and manually opt-in is required, such as by setting
+  <a href="#v-">`server.enabled`</a> to true.
+
+
+  - <a name="v-global-domain" href="#v-global-domain">`domain`</a> (`string: "consul"`) -
+  The domain Consul uses for DNS queries. This is used to configure agents both
+  for DNS listening but also to know what domain to join to join the cluster.
+  This should be consistent throughout the chart, but can be overridden
+  per-component as well.
+
+  - <a name="v-global-image" href="#v-global-image">`image`</a> (`string: "consul:latest"`) -
+  The name of the Docker image (including any tag) for the containers running
+  Consul agents.
```

Comment:
Does this mean that reconfiguring some other feature in consul will also potentially upgrade to a newer Consul version unless this is pinned to something specific?

If so, it feels like a clearer warning about that should be included here - we already said these might not be good production values but it seems like you'd never want to just let helm pick whichever version o fconsul is latest when it runs in prod?

Reply:
Yes, let me warn.